compiler/src/dotty/tools/dotc/quoted/QuoteDriver.scala

```diff
@@ -11,7 +11,8 @@ import scala.quoted.{Expr, Type}
 
 import java.net.URLClassLoader
 
-import Toolbox.{Settings, Run, Show}
+import Toolbox.{Run, Settings, Show}
+import dotty.tools.dotc.core.Definitions
 
 class QuoteDriver extends Driver {
```

Comment:
Since you're always using QuoteDriver from an enclosing compiler with its own context, why not get the definitions from the enclosing ctx.definitions?

Reply:
Enclosing compiler?

Reply:
Quotes are expanded by some compiler, this compiler has a context with definitions.